cube/src/main/java/zone/cogni/asquare/cube/convertor/ModelToJsonConversion.java

```diff
@@ -268,16 +269,19 @@ private void reportUnprocessedTriples(Context context, String root) {
     }
   }
 
+  public Model getMissedTriples() {
+    if (context.alreadyProcessedModel == null) throw new RuntimeException("Can't fetch missed triples: model to json hasn't run yet");
+    return context.model.difference(context.alreadyProcessedModel);
```

Comment:
not even sure if this will work correctly. we recently found Jena has issues with typed literals outside of the default namespaces as xsd or rdfs